src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/devices-page.mdx

```diff
@@ -1,65 +1,67 @@
 ---
-title: Devices page
+title: 'Devices page: Compare app performance between devices'
```

Comment:
I THINK that compare isn't supposed to be capitalized even with the colon but I'm not 100%. If you know I'm wrong then for sure ignore this comment!